clientv3/integration/network_partition_test.go

```diff
@@ -310,10 +310,20 @@ func TestDropReadUnderNetworkPartition(t *testing.T) {
 		t.Fatalf("expected %v, got %v", rpctypes.ErrLeaderChanged, err)
 	}
 
-	ctx, cancel = context.WithTimeout(context.TODO(), 10*time.Second)
-	_, err = kvc.Get(ctx, "a")
-	cancel()
-	if err != nil {
-		t.Fatalf("expected nil, got %v", err)
+	retry := true
+	i, maxRetry := 0, 5
+	for retry && i < maxRetry {
+		ctx, cancel = context.WithTimeout(context.TODO(), 10*time.Second)
+		_, err = kvc.Get(ctx, "a")
+		cancel()
+		if err != nil {
+			if err == rpctypes.ErrTimeout {
+				<-time.After(time.Second)
+				i++
+				continue
+			}
+			t.Fatalf("expected nil or timeout, got %v", err)
+		}
+		retry = false
```

Comment:
use break instead to break out of the loop? @johncming

Reply:
I'm not sure what its means, can you explain more? thanks.

Reply:
ok.